tasks/Ubuntu.yml

```diff
@@ -16,6 +16,14 @@
     loop_var: repo
   no_log: true
 
+- name: "Install required packages to run add-apt-repository"
+  package:
+    name: "software-properties-common"
+    state: present
+
+- name: "Configure `universe` repositories for {{ ansible_lsb.id }} {{ ansible_distribution_release }}, required for python-software-properties"
+  command: 'add-apt-repository universe'
```

Comment:
This command fails the idempotency test.
Maybe it would be better to use the apt_repository module ?
http://docs.ansible.com/ansible/latest/modules/apt_repository_module.html